nix/derivation.nix

```diff
@@ -60,9 +62,9 @@ in {
 
   shell = {
     buildInputs = unique ([
+      yarn
```

Comment:
Why did yarn need to come first in the list?

Reply:
OCD